configuration/doc/README.md

```diff
@@ -27,11 +27,121 @@ plain backend, which only tracks calls to the compiler, but does not act upon
 them. An example invocation for the plain backend and a project whose build
 command is "make" would be:
 
- one-line-scan --plain --no-gotocc -- make
+  one-line-scan --plain --no-gotocc -- make
 
-More invocations of one-line-scan are presented in the file
-configuration/doc/EXAMPLES.md
+The available parameters of the tool can be seen when using the --help flag:
+
+  one-line-scan --help
+
+More invocations of one-line-scan are presented in the file doc/EXAMPLES.md
+
+By default, the wrapper for CBMC is enabled. Furthermore, analysis is enabled by
+default. To disable CBMC, use --no-gotocc, and to disable analysis, use the
+parameter --no-analysis.
```

Comment:
Just a thought: should we actually make "plain" the default?

Reply:
Yes, that would work for me.